tests/objects/test_bitmask.py

```diff
@@ -78,7 +80,15 @@ def test_mask_rle_mask_roundtrip():
     assert np.allclose(orig_mask, new_mask)
 
 
-def test_transpose_bytearray():
+def test_transpose_bytearray_python():
```

Comment:
These two are identical.
Can we directly import the two different implementations inside the test? Should always have numpy in CI and imports inside test won't be run otherwise

Reply:
Ah, good spot, yes, will clean it up